docs/modules/user-guide/partials/proc_migrating-components.adoc

```diff
@@ -2,13 +2,11 @@
 = Migrating components
 
 [role="_abstract"]
-This section describes how to migrate components to devfile v2. There ARE major changes in the `components` section of the devfile. A component specified in a v1.0 devfile will NOT work as it is in a devfile v2.0.
+This section describes how to migrate components to devfiles v2.x. The components section of devfiles v2.x has major changes. A component specified in a devfile v1.x no longer works in a devfile v2.x.
```

Comment:
"differs considerably from devfile v1.0"?
The formulation "has major changes" doesn't provide any context. "A file has changes" doesn't tell in comparison to what it changed, also it is active voice, but in this case I would advocate for a different change of words or even passive voice. I think "Major changes have been made to the components section in devfile v2.x" is easier to understand.